www/netlify.toml

```diff
@@ -0,0 +1,6 @@
+[build]
```

Comment:
@eliwilliamson this file needs to be in the root of the repository. It also has to include the next line under the `build` section to ensure we build from the www directory:

```
base = "www"
```